Revit_Engine/Create/Config/RevitPullConfig.cs

```diff
@@ -32,7 +32,9 @@ public static partial class Create
         /***************************************************/
         /****              Public methods               ****/
         /***************************************************/
-        
+
+        [PreviousVersion("4.3", "BH.Engine.Adapters.Revit.Create.RevitPullConfig(BH.oM.Adapters.Revit.Enums.Discipline, System.Boolean, BH.oM.Adapters.Revit.PullGeometryConfig, BH.oM.Adapters.Revit.PullRepresentationConfig, System.Boolean)")]
+        [PreviousVersion("4.3", "BH.Engine.Adapters.Revit.Create.RevitPullConfig(BH.oM.Adapters.Revit.Enums.Discipline, System.Boolean, BH.oM.Adapters.Revit.PullGeometryConfig, BH.oM.Adapters.Revit.PullRepresentationConfig)")]
```

Comment:
Do you need both of these attributes? Sorry, I'm far from a Versioning expert, but this isn't something I've seen.

Reply:
I think the purpose of versioning is to try and fix for multiple versions going backwards. If prior to this version both were used and it now wants to upgrade from 3.3 to 4.3 and 4.0 to 4.3 in the same manner, then both attributes would be acceptable.

We've used it in the Engine a couple of times when moving a method from Environment_Engine and Structure_Engine into a common engine such as Analytical_Engine (to work on the generic panel), we have then used `PreviousVersion` to indicate both previous engine methods can now version into this new method.

Obviously that's taking 2 methods into 1 from the same version, this is similar only in that it is taking 2 methods from different versions into 1 for the current version. Based on my understanding of the issue at least. So could be wrong with what's needed, but certainly from a Versioning perspective, 2 attributes like this would be valid.

Reply:
Yeah this is exactly what is happening @FraserGreenroyd, 2 previous versions of the method not versioned but simply deprecated and removed, now turning red or magic on the canvas. Looks like we teamed up quite well: I knew what is needed, you confirmed that my 'give it a shot' solution makes sense 😃 Thanks both!